### ISSUE TITLE
Add package for pywal 

### ISSUE THREAD

**Issue by jalfonsi** [regolith-linux/speed-ricer#2]
https://github.com/dylanaraps/pywal

**Comment by jalfonsi** [comment]
Oomox project already provide debs, maybe porting is easier and needs only updating to Buster release
https://github.com/themix-project/oomox/releases

**Comment by kgilmer** [comment]
Good call!  I'll look into it.  The oomox link you provided is very helpful.

**Comment by kgilmer** [comment]
@jalfonsi I have not used pywal before.  The package is now staged.  Any chance you can test it?

**Comment by jalfonsi** [comment]
Ok, I'll see what I can do.

**Comment by jalfonsi** [comment]
Another repo with a list of interesting tools. See "tweaks" section
https://github.com/myugan/awesome-linux-customization

**Comment by kgilmer** [comment]
Wow, some cool projects here.  Thanks for sending this to me Jessica!

-ken
